### ISSUE TITLE
v2.0.0 no longer support PHP 7.3?

### ISSUE THREAD

**Issue by mzch** [shlinkio/shlink#606]
#### How Shlink is set-up

* Shlink Version: 2.0.0
* PHP Version: 7.3.11-1~deb10u1
* How do you serve Shlink: Self-hosted Apache (FPM)
* Database engine used: MariaDB

#### Summary
when doing `bin/update`, an error occurred.

#### Current behavior
```
PHP Parse error:  syntax error, unexpected ':', expecting ';' in /srv/shlink_2.0.0_dist/vendor/shlinkio/shlink-installer/src/Factory/SwooleInstalledFactory.php on line 15
```

#### Expected behavior
finished updating normally.

#### How to reproduce
Using PHP 7.3.x

**Comment by acelaya** [comment]
Yes, Shlink 2.x requieres PHP 7.4, it's not a bug.

**Comment by alfiepates** [comment]
This doesn't seem to be documented in the release notes. 

If 2.x is not going to support PHP 7.3, will security fixes be backported to 1.21.x to support those of us on Debian stable?

**Comment by acelaya** [comment]
> This doesn't seem to be documented in the release notes.

It's actually the first point both in the release notes and the upgrading documentation.

* https://github.com/shlinkio/shlink/releases/tag/v2.0.0
* https://github.com/shlinkio/shlink/blob/develop/UPGRADE.md#php-74-required

> If 2.x is not going to support PHP 7.3, will security fixes be backported to 1.21.x to support those of us on Debian stable?

It will be decided on a per-issue basis, but I'm afraid I cannot warranty it, as I don't have the capacity.
